### PR TITLE
Fix https://github.com/wso2/product-apim/issues/2461

### PR DESCRIPTION
**Pull request by Krishanx92**
### Proposed changes in this pull request

Fix https://github.com/wso2/product-apim/issues/2461

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js [hunk 1]
```diff
@@ -322,7 +334,37 @@ class BasicInfo extends Component {
...
-                                    <IconButton aria-label="Share">
+                                    <IconButton aria-label="Share" onClick={this.handlePopupOpen}>
+                                        <Popover
+                                            open={this.state.openPopup}
+                                            anchorEl={this.state.anchorEl}
+                                            onClose={this.handlePopupClose}>
+                                            <div id="share_div_social" className="share_dives">
+                                                {/* Facebook */}
+                                                <a className="social_links" id="facebook"
+                                                   href="http://www.facebook.com/sharer.php?u=https%3A%2F%2F172.17.0.1%3A9444%2Fstore%2Fapis%2Finfo%3Fname%3Dfoo%26version%3D1.0.0%26provider%3Dadmin"
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js]: sharing URL cannot be static. Need to map correct URLs

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js [hunk 2]
```diff
@@ -322,7 +334,37 @@ class BasicInfo extends Component {
...
+                                                   target="_blank" title="facebook">
+                                                    <img src="/store/public/app/images/social/facebook.png" alt="Facebook" />
+                                                </a>
+                                                {/* Twitter */}
+                                                <a className="social_links" id="twitter"
+                                                   href="http://twitter.com/share?url=https%3A%2F%2F172.17.0.1%3A9444%2Fstore%2Fapis%2Finfo%3Fname%3Dfoo%26version%3D1.0.0%26provider%3Dadmin&text=API%20Store%20-%20foo%20%3A%20try%20this%20API%20at%20https%3A%2F%2F172.17.0.1%3A9444%2Fstore%2Fapis%2Finfo%3Fname%3Dfoo%26version%3D1.0.0%26provider%3Dadmin"
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js]: sharing URL cannot be static. Need to map correct URLs

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js [hunk 3]
```diff
@@ -322,7 +334,37 @@ class BasicInfo extends Component {
...
+                                                   target="_blank" title="twitter">
+                                                    <img src="/store/public/app/images/social/twitter.png" alt="Twitter" /></a>
+                                                {/* Google+ */}
+                                                <a className="social_links" id="googleplus"
+                                                   href="https://plus.google.com/share?url=https%3A%2F%2F172.17.0.1%3A9444%2Fstore%2Fapis%2Finfo%3Fname%3Dfoo%26version%3D1.0.0%26provider%3Dadmin"
+                                                   target="_blank" title="googleplus">
+                                                    <img src="/store/public/app/images/social/google.png" alt="Google" /></a>
+                                                {/* Digg */}
+                                                <a className="social_links" id="digg"
+                                                   href="http://www.digg.com/submit?url=https%3A%2F%2F172.17.0.1%3A9444%2Fstore%2Fapis%2Finfo%3Fname%3Dfoo%26version%3D1.0.0%26provider%3Dadmin"
```
Comment (dakshika) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js]: sharing URL cannot be static. Need to map correct URLs

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js (hunk 1 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js]: This is fix with: https://github.com/wso2/carbon-apimgt/pull/5284/

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js (hunk 2 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js]: This is fix with: https://github.com/wso2/carbon-apimgt/pull/5284/

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js (hunk 3 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/BasicInfo.js]: This is change with https://github.com/wso2/carbon-apimgt/pull/5284/

**Pull request merged**